src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAddMovie: (newMovie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie>();
```

Comment:
```suggestion
  const [movie, setMovie] = useState<Movie | null>(null);
```
use your value or null